views/components/progress_bar.erb

```diff
@@ -0,0 +1,10 @@
+<% progress = [100, 100 * numerator / denominator].min %>
+<% color = (progress < 60) ? "bg-blue-500" : (progress < 80) ? "bg-yellow-500" : "bg-red-500" %>
```

Comment:
I was just as confused as Furkan. IMHO, it's better to use a single common orange color rather than multiple ones. However, this is purely subjective.

Reply:
I think the reason of the confusion was that I tried to show-case all 3 possible states in the screenshot. In real world, most of them would be blue and 1 yellow/red would be less confusing. I prefer to keep it this way for now.